sechub-wrapper-xray/src/main/java/com/mercedesbenz/sechub/wrapper/xray/util/ZipFileCreator.java

```diff
@@ -0,0 +1,23 @@
+package com.mercedesbenz.sechub.wrapper.xray.util;
+
+import java.io.*;
+
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
+import com.mercedesbenz.sechub.wrapper.xray.cli.XrayWrapperExitCode;
+
+public class ZipFileCreator {
+    public void zip(File file, InputStream inputStream) throws XrayWrapperException {
+        int read;
+        byte[] buffer = new byte[1024];
+        FileOutputStream fileOutputStream = null;
+        try {
```

Comment:
Please use auto close:
```suggestion
        try (FileOutputStream fileOutputStream = new FileOutputStream(file){
        
```
You do not need to handle inputstream close again and it does also handle closing of the stream in case of an io exception